google/cloud/bigtable/internal/rate_limiter.cc

```diff
@@ -0,0 +1,80 @@
+// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/bigtable/internal/rate_limiter.h"
+#include "absl/time/time.h"
+
+namespace google {
+namespace cloud {
+namespace bigtable_internal {
+GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
+namespace {
+
+using Clock = RateLimiter::Clock;
+
+/**
```

Comment:
s/**/*/

Reply:
why?

I know `/**` is a doxygen / javadoc thing. I find it simpler to write comments the same way for public and private APIs.

Reply:
I think we use `/**` more often than `/*` for internal documentation. I am not going to change these.

> it is better to make things easier to read than to write

Agreed. Don't think it applies here.